packages/hero/src/scss/_component.scss

```diff
@@ -25,8 +25,12 @@
 
   &__text {
     
-    & h1 {
+    & h1,
+    & .uq-hero__title {
+
       color: inherit;
+      font-family: core.$montserrat;
```

Comment:
Fonts were not loading consistently for me, we need to check against the actual font that it is working.